### PR TITLE
[refactor] improve readability

### PR DESCRIPTION
**Pull request by kalikaneko**
Improve readability of operations on generic keys, by assigning the
class matching the type of key (_wrapper_map[ktype]) at the beginning of
each block.

in the future, we could pass the type of key (only PGP keys being used
at the moment) on initialization of the Keymanager, so we don't have to
pass the ktype on each method call.

### CODE REVIEW

**Comment by kalikaneko** [comment]
uh, I think this is breaking something :( on hold...

**Comment by kalikaneko** [comment]
should be good now. it turns out that the "keys" choice wasn't very good after all.
I'm concerned that tests didn't pick these errors.

**Comment by meskio** [comment]
Looks good, rebase it and I'll merge it.

File: src/leap/keymanager/__init__.py [hunk 1]
```diff
@@ -545,14 +548,15 @@ def encrypt(self, data, address, ktype, passphrase=None, sign=None,
...
 
         def encrypt(keys):
             pubkey, signkey = keys
-            encrypted = self._wrapper_map[ktype].encrypt(
+            encrypted = _keys.encrypt(
                 data, pubkey, passphrase, sign=signkey,
                 cipher_algo=cipher_algo)
             pubkey.encr_used = True
-            d = self._wrapper_map[ktype].put_key(pubkey, address)
+            d = keys.put_key(pubkey, address)
```
Comment (meskio) [inline src/leap/keymanager/__init__.py]: s/keys/_keys/

**Comment by kalikaneko** [comment]
force-pushed, rebased

**Pull request merged**